core/context.go

```diff
@@ -102,7 +104,7 @@ func (ctx *Context) Close() {
 	log.Info("shutting down IPVS context")
 
 	// This will also shutdown the pulse notification sink goroutine.
-	close(ctx.pulseCh)
+	close(ctx.stopCh)
```

Comment:
You close pulseCh, but Loop goroutine will try to send an update to it. It's panic. As there's no way to find out if the channel is closed, you cannot close it or make sure that no one will use the channel for writting after close.